docs/joinrun.md

```diff
@@ -242,11 +255,18 @@ join(
 join(
   run { case c(x) + i(_) => c(x+1) }
 ) // runtime error: "c" is already bound to another join definition
+
 ```
 
-This rule enforces the immutability of chemical laws: it is impossible to add a new reaction that consumes a molecule already declared as input to a previous reaction.
+This rule enforces the immutability of chemical laws:
+Once a join definition is written, we fix the reactions that a given molecule could initiate (i.e. the reactions that will consume this molecule).
```

Comment:
But if so, I would like to rewrite the entire tutorial, emphasizing "reaction sites" and the visual image of molecules going to the reaction site in order to find their friends.